elide-contrib/dropwizard-elide/src/main/java/com/yahoo/elide/contrib/dropwizard/elide/ElideBundle.java

```diff
@@ -0,0 +1,135 @@
+import org.glassfish.jersey.server.internal.scanning.AnnotationAcceptingListener;
+import org.glassfish.jersey.server.internal.scanning.PackageNamesScanner;
+import org.hibernate.SessionFactory;
+
+import javax.persistence.Entity;
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Elide Bundle
+ *
+ * @param <T> Dropwizard Configuration
+ */
+public abstract class ElideBundle<T extends Configuration>
+        implements ConfiguredBundle<T>, DatabaseConfiguration<T>, ElideConfiguration<T> {
+
+    public static final String DEFAULT_NAME = "elide-bundle";
+
+    private final ImmutableList<Class<?>> entities;
+    private final SessionFactoryFactory sessionFactoryFactory;
+
+    /**
+     * @param pckg string with package containing Hibernate entities (classes annotated with Hibernate {@code @Entity}
+     *             annotation) e. g. {@code com.codahale.fake.db.directory.entities}
+     */
+    protected ElideBundle(String pckg) {
+        this(pckg, new SessionFactoryFactory());
+    }
+
+    protected ElideBundle(String pckg, SessionFactoryFactory sessionFactoryFactory) {
+        this(new String[] { pckg }, sessionFactoryFactory);
+    }
+
+    protected ElideBundle(String[] pckgs, SessionFactoryFactory sessionFactoryFactory) {
+        this(findEntityClassesFromDirectory(pckgs), sessionFactoryFactory);
+    }
+
+    public ElideBundle(Class<?> entity, Class<?>... entities) {
+        this(ImmutableList.<Class<?>>builder().add(entity).add(entities).build(),
+                new SessionFactoryFactory());
+    }
+
+    public ElideBundle(ImmutableList<Class<?>> entities,
+                       SessionFactoryFactory sessionFactoryFactory) {
+        this.entities = entities;
+        this.sessionFactoryFactory = sessionFactoryFactory;
+    }
+
+    protected String name() {
+        return DEFAULT_NAME;
+    }
+
+    @Override
+    public void initialize(Bootstrap<?> bootstrap) {
+    }
+
+    @Override
+    public void run(T configuration, Environment environment) throws Exception {
+        final AuditLogger auditLogger = getAuditLogger(configuration, environment);
+        final DataStore dataStore = getDataStore(configuration, environment);
+        final JsonApiEndpoint.DefaultOpaqueUserFunction getUserFn = getUserFn(configuration, environment);
+
+        environment.jersey().register(new AbstractBinder() {
+            @Override
+            protected void configure() {
+                bind(getUserFn)
+                        .to(JsonApiEndpoint.DefaultOpaqueUserFunction.class)
+                        .named("elideUserExtractionFunction");
+                bind(new Elide.Builder(dataStore).auditLogger(auditLogger).build()).to(Elide.class).named("elide");
+            }
+        });
+    }
+
+    protected void configure(org.hibernate.cfg.Configuration configuration) {
+    }
+
+    @Override
+    public DataStore getDataStore(T configuration, Environment environment) {
```

Comment:
Actually, I see the interface below. It looks as though it's pretty straightforward to override what would be necessary.